manifests/gcp_marketplace/hack/release.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+#
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -ex
+
+TAG_NAME=$1
+DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" > /dev/null && pwd)"
+
+if [[ -z "$TAG_NAME" ]]; then
+  echo "Usage: release.sh <release-tag>" >&2
+  exit 1
+fi
+
```

Comment:
would you add a README somewhere to mention how to use these hack/release.sh for a release.
Although Googler will handle the release but better external know it via doc for better mantainence.

I actually didn't follow via just read the codes in this PR.

E.x. how to release a 1.0.0.0 from the branch. (or 1.0.0-RC.0 if MKP allowed, I sent an email and cc'ed you)

Reply:
Thanks! I'll add a short README together with this PR.

Reply:
thanks. would be happy to take a try via following the README

Reply:
for the pattern on whether MKP can support the 3 dots version string, let's wait for Erika reply.
It's better we can put RC to MKP test version (kubeflow-pipelines-test) for internal test.

Reply:
totally agree, if that's not supported we may use a different versioning convention to make that possible